### ISSUE TITLE
Build problems in WSL environment: file rename permission denied (formerly: Latest Binaries for Ubuntu 20.04 GCC 11.4 version)

### ISSUE THREAD

**Issue by skothala09** [docwire/docwire#160]
Hi Team,

As I am facing the issues for compilation of the latest code, please share the latest version binaries for the library for Ubuntu 20.04, GCC version 11.4. 
As I haven't found in the latest tags. And also help in fixing the build issues.
May be a step by step document will be really helpful. 
Thank you in advance

**Comment by skothala09** [comment]
![image](https://github.com/user-attachments/assets/395160ca-e1bb-4c16-bf25-4157c895cf24)
And also facing the issue while using the library that has been shared earlier

**Comment by as-ascii** [comment]
Thank you for reporting the error. Could you please attach log files related to building error so we can help to fix it? Name of related log files are usually printed by vcpkg in the error message.

Regarding the runtime error: could you please check it with docwire CLI  utility from your installation? Its path is "vcpkg/installed/x64-linux-dynamic/debug/tools/docwire" Please check with and without "--use-stream 1"  option to check detecting by file extension vs file content.

**Comment by skothala09** [comment]
![image](https://github.com/user-attachments/assets/8d8efa42-81df-4b0a-950a-e18040336082)
This is the error I am getting, I have tried multiple times

**Comment by as-ascii** [comment]
Hello. It seems that the problem is with file permissions but it's strange because command is executed as "root" user as we can see. What is the filesystem in /mnt/d ? Is this a share drive? You can check it using "df" command for example. Could you please check if it is possible to rename or remove the reported file (ratio-boost-1.86.0) manually from command line? We would like to check if this is a vcpkg issue that should be reported upstream or a local problem that we can resolve together.

**Comment by as-ascii** [comment]
It can be a problem similar to https://github.com/docwire/docwire/issues/138

**Comment by as-ascii** [comment]
Similar upstream issue in WSL: https://github.com/microsoft/vcpkg/issues/27527
Is this a WSL installation maybe?

**Comment by skothala09** [comment]
Yes I am trying to build in WSL environment. /mnt/d/ is not a shared drive
![image](https://github.com/user-attachments/assets/705d6a44-006f-46ad-92b1-99ba162c19f4)

**Comment by skothala09** [comment]
> --use-stream 1

![image](https://github.com/user-attachments/assets/b3690703-750a-425d-b904-1207fe35714
I have tested using  the mentioned path still it is not working

**Comment by as-ascii** [comment]
> > --use-stream 1
> 
> ![image](https://github.com/user-attachments/assets/b3690703-750a-425d-b904-1207fe35714 I have tested using the mentioned path still it is not working

Could you please attach problematic file or is it confidential?

**Comment by as-ascii** [comment]
> Yes I am trying to build in WSL environment. /mnt/d/ is not a shared drive ![image](https://private-user-images.githubusercontent.com/158155620/384836073-705d6a44-006f-46ad-92b1-99ba162c19f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEzNTU4MzcsIm5iZiI6MTczMTM1NTUzNywicGF0aCI6Ii8xNTgxNTU2MjAvMzg0ODM2MDczLTcwNWQ2YTQ0LTAwNmYtNDZhZC05MmIxLTk5YmExNjJjMTlmNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExMVQyMDA1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05Y2E5YzFlZjJjOGIwNWY2MGM4MjRhYjhhOGI0NTlmYTE4ZDA0NWUwYTZkZjg5ZjZhZTdkNDExNTQyNjI0ZjUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ETrVf_jcTw5NcMjrlOQYcA6rESqYThKk7V729HEewKA)

It seems to be logical that antivirus can be a problem as it can lock newly created files for scanning and it does not know that there are a temporary files removed or moved quickly. Could you please try to disable antivirus while building packages or reconfigure it to ignore the vcpkg directory?

The binaries can be delivered as a workaround but the problem will still exists than with every fix or upgrade.

**Comment by as-ascii** [comment]
Related issues in upstream (vcpkg):
https://github.com/microsoft/vcpkg/issues/11283
https://github.com/microsoft/vcpkg/issues/12204

**Comment by as-ascii** [comment]
> please share the latest version binaries for the library for Ubuntu 20.04, GCC version 11.4

Please find the binaries here as a temporary workaround:
https://github.com/docwire/docwire/actions/runs/11840115132/artifacts/2188431371
There were produced with "vcpkg export":
https://learn.microsoft.com/en-us/vcpkg/commands/export

**Comment by as-ascii** [comment]
Closing this, as upstream issues are closed and in addition Ubuntu 20.04 is not supported by DocWire any more (after GitHub deprecated the runners).
